crates/nebula-backbone/src/domain/secret/mod.rs

```diff
@@ -636,9 +649,46 @@ fn validate_path(path: &str) -> Result<()> {
 
 pub(crate) struct PostgresSecretService {}
 
+fn check_secret_accessible(secret_policy: &policy::Model, claim: &NebulaClaim) -> bool {
+    let (parsed_policy, _) =
+        nebula_policy::pest::parse(&secret_policy.expression, nebula_policy::pest::PolicyLanguage::HumanPolicy)
+            .expect("registered policy should be parsable");
```

Comment:
I suggest handling this error case more gracefully instead of using `.expect()`. 
The current `.expect()` will cause the program to panic, which isn't ideal for production code. A more robust error handling approach would improve reliability and maintainability.

Reply:
I have modified it so that an error occurs if an issue arises during the process of interpreting the policy.